packages/dds/matrix/src/matrix.ts

```diff
@@ -546,13 +612,19 @@ export class SharedMatrix<T = any>
 				);
 
 				const setOp = content as ISetOp<T>;
-				const { rowHandle, colHandle, localSeq, rowsRefSeq, colsRefSeq } =
-					localOpMetadata as ISetOpMetadata;
-
-				// If there are more pending local writes to the same row/col handle, it is important
-				// to skip resubmitting this op since it is possible the row/col handle has been recycled
-				// and now refers to a different position than when this op was originally submitted.
-				if (this.isLatestPendingWrite(rowHandle, colHandle, localSeq)) {
```

Comment:
I'm not sure this existing comment and old logic makes sense. If I got it right, it essentially skips all the changes and leaves only latest change when doing rebasing. But I believe it's wrong - even though from final state all intermediate changes do not matter, they do matter in our current model where we expose all intermediate states (via raising change events) and ensure that all such states would be exposed without any kind of reordering or skipping.


Here are couple things that do not add up for me:
1. sendSetCellOp() does not have similar check. Resubmit flow is somewhat similar. Why do we have to be different?
2.